identity/src/main/java/com/android/identity/PresentationHelper.java

```diff
@@ -917,13 +918,21 @@ byte[] nfcEngagementHandleSelectFile(@NonNull byte[] apdu) {
 
     private @NonNull
     byte[] nfcEngagementHandleReadBinary(@NonNull byte[] apdu) {
-        mLog.session("in nfcEngagementHandleReadBinary");
         if (apdu.length < 5) {
             return STATUS_WORD_FILE_NOT_FOUND;
         }
         byte[] contents = mSelectedNfcFile;
         int offset = (apdu[2] & 0xff) * 256 + (apdu[3] & 0xff);
         int size = apdu[4] & 0xff;
+        if (size == 0) {
+            // Handle Extended Length encoding
```

Comment:
A pointer to a specification would be nice.

However, if the spec is not open and we cannot easily link to it, then maybe this explanation is the best we can do.

Reply:
It's not publicly available, I'm afraid. I think this suffices, the concept of extended field is pretty fundamental.